projects/addon-editor/components/editor-new/editor-new.component.ts

```diff
@@ -65,6 +58,8 @@ export class TuiEditorNewComponent
     @ViewChild(TuiToolbarNewComponent)
     readonly toolbar?: TuiToolbarNewComponent;
 
+    focused = false;
```

Comment:
we have focused field in AbstractTuiControl, can't we use it ?

Reply:
It's abstract there, we typically implement it as getter but in this case we have to manually manage it.